src/bramble/client/main.js

```diff
@@ -953,7 +953,7 @@ define([
     BrambleProxy.prototype.export = function(callback) {
         this._executeRemoteCommand({commandCategory: "bramble", command: "BRAMBLE_EXPORT"}, callback);
     };
-	
```

Comment:
You didn't fix this.  Please fix all review issues before requesting another review.  There's no point going over the same mistakes multiple times in reviews.